gaia-sdk-java/gaia-sdk-graphql/src/main/kotlin/gaia/sdk/GaiaClientBuilder.kt

```diff
@@ -1,33 +1,34 @@
 package gaia.sdk
 
-import gaia.sdk.client.HMAC
 import gaia.sdk.spi.ClientOptions
 import gaia.sdk.spi.ITransporter
 
-class GaiaClientBuilder (private val transporter: ITransporter) {
+//Class generated from template src/main/resources/template/java/ClientBuilderTemplate.vm
 
-    lateinit var apiKey: String;
-    lateinit var secret: String;
-    lateinit var bearer: String;
+class GaiaClientBuilder (private val transporter: ITransporter) {
 
-    fun withApiKey(apiKey: String): GaiaClientBuilder {
-        this.apiKey = apiKey
-        return this
-    }
+    lateinit var credentials: GaiaCredentials
+    var contentType: String = "application/json"
 
-    fun withSecret(secret: String): GaiaClientBuilder {
-        this.secret = secret
+    fun withCredentials(credentials: GaiaCredentials): GaiaClientBuilder {
+        this.credentials=credentials
         return this
     }
 
-    fun withBearer(bearer: String): GaiaClientBuilder {
-        this.bearer = bearer
+    fun withContentType(contentType: String) : GaiaClientBuilder {
```

Comment:
Sollte der Content Type so statisch gesetzt werden? Sollte den der User überhaupt definieren können?

Reply:
Aus meiner sicht ist es derzeit sinnloss. Es wird immer application/json verwendet. Aber Chris hat es zu dem ClientOptions gegeben und ClientOptions werden eigentlich durch GaiaClientBuilder instanzieren.D.h. wenn ich beim Builder keine "With contentType" anbiete, dann koennen wir schon löschen.
Ich weiss net ob in der Zukunft unterschiedliche ContentTypes verwenden sollten.

Reply:
Alles klar, dann lassen wir es derweil